backend/Dockerfile

```
@@ -1,19 +1,16 @@
-FROM strapi/base:alpine
-
-ARG STRAPI_VERSION
-RUN yarn global add @strapi/strapi@${STRAPI_VERSION}
-
-WORKDIR /srv/app
-
-COPY ./package*.json ./
-RUN npm install
-
-COPY ./src/ ./src/
-COPY ./config/ ./config/
-COPY ./database/ ./database/
-COPY ./public/ ./public/
-COPY ./favicon.ico ./
-
-RUN npm run build
-
-CMD ["npm", "start"]
\ No newline at end of file
+FROM node:16-alpine
+# Installing libvips for sharp compatibility
+RUN apk update && apk add vips-dev
```

You can skip the `update` with the `-U/--update-cache` command. And once that's complete, you can clear out the APK cache in that layer to lighten up the docker image:
```suggestion
RUN apk --update-cache add vips-dev \
 && rm -rf /var/cache/apk/*
```